### ISSUE TITLE
Background color

### ISSUE THREAD

**Issue by unformatt** [facelessuser/ExportHtml#38]
> Background colors are disabled in browser printing by default, so they will not print until background color printing is enabled

Does this mean until I enable "background color printing" somehow or this is a future feature?

**Comment by facelessuser** [comment]
Web browsers often don't print background colors of web pages. This is a sensible thing as it is likely you don't want to waste all your blue ink just because the website course to have a blue background.   Some browsers can turn this off,  but it is quite likely this isn't what you want to do.   Most likely,  you want to prove with a white background.

**Comment by unformatt** [comment]
Ah, so it means "so they will not print until background color printing is enabled [in web browsers]"

Well, makes sense in that case! But I'm actually using this so I can paste some pretty code into an email (or a WYSIWYG) - so I guess I'm coming at it from the wrong angle!

**Comment by facelessuser** [comment]
You can copy and paste in email fine (depending on through what email client you send it through).  I use it in outlook at work and gmail etc.

**Comment by unformatt** [comment]
OK - so I realized we're not talking about the same thing. When I exported, I was expecting a dark background that matched my SublimeText settings but I'm now realizing export does not match ST theme but works from tmThemes. So with my incorrect notion, I checked the README and I was thinking in the wrong context about backgrounds - I thought it was saying that exported HTML didn't support background color. Thanks for humoring me!

![](http://i.imgur.com/rKoiOXD.png)
